### PR TITLE
Add IBM Copyright Statements

### PR DESCRIPTION
**Pull request by chrisdennis**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ////
   // Copyright 2015-2023 Terracotta, Inc., a Software AG company.
+  // Copyright Super iPaaS Integration LLC, an IBM Company 2024
```
Comment (cljohnso) [inline README.adoc:3]: Appear to have missed one below -- `(C) Terracotta, Inc., a Software AG company.` lacks the Super iPaaS variant.

**Comment by cljohnso** [comment]
This project has IntelliJ IDEA copyright profiles committed -- they should be updated or removed.

**Comment by cljohnso** [comment]
Several files contain a variant of `(all content copyright.* Terracotta, Inc.)` _in addition to_ the normal copyright/license.  Should these remain?

**Review by cljohnso — CHANGES_REQUESTED** [review]
At least one file needs a change; a few others _may_ need a change.

**Review by cljohnso — CHANGES_REQUESTED** [review]
Several files contain a variant of `(all content copyright.* Terracotta, Inc.)` in addition to the normal copyright/license. Should these remain?

**Comment by chrisdennis** [comment]
> Several files contain a variant of `(all content copyright.* Terracotta, Inc.)` _in addition to_ the normal copyright/license. Should these remain?

Remnant of the reintegration of OSS and EE offheap fragments. Will delete them.

**Review by cljohnso — APPROVED** [review]

**Pull request merged**